app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncMessage.java

```diff
@@ -0,0 +1,142 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_message")
+public class SenderSyncMessage extends AbstractEntity {
+	
+	private static final long serialVersionUID = 1L;
+	
+	// TODO add ACKNOWLEDGED status, which will be set after successful processing of the receiver response
+	public enum SenderSyncMessageStatus {
+		NEW,
+		SENT,
+		ERROR
```

Comment:
The ticket does not say ERROR is one of the statuses, please remove it

Reply:
Removed.